corehq/util/timezones/tests/test_utils.py

```diff
@@ -0,0 +1,37 @@
+import pytz
+from mock import patch
+
+from django.test import SimpleTestCase, override_settings
+
+from corehq.apps.users.models import WebUser, DomainMembership
+from corehq.util.timezones.utils import get_timezone_for_user
+
+DOMAIN_TIMEZONE = pytz.timezone('Asia/Kolkata')
+
+
+@patch('corehq.util.timezones.utils.get_timezone_for_domain', lambda x: DOMAIN_TIMEZONE)
+@patch('corehq.apps.users.models._AuthorizableMixin.get_domain_membership')
+class GetTimezoneForUserTest(SimpleTestCase):
+    def test_no_user(self, _):
+        self.assertEqual(get_timezone_for_user(None, "test"), DOMAIN_TIMEZONE)
+
+    def test_user_with_no_domain_membership(self, domain_membership_mock):
+        couch_user = WebUser()
+        domain_membership_mock.return_value = None
+        self.assertEqual(get_timezone_for_user(couch_user, "test"), DOMAIN_TIMEZONE)
```

Comment:
if the user isn't a member of the domain, why do they get the domain timezone?  Shouldn't they get `settings.TIME_ZONE`?

Reply:
That has been the implementation so can't say for sure.
What i can think of is, Since HQ views are almost always bound to a domain like you are always viewing data for a domain, falling back to domain timezone sounds reasonable. The word "global" here, the one used as a boolean, I think is misleading.